src/extensionGlobals.ts

```diff
@@ -0,0 +1,21 @@
+import { ExtensionContext } from "vscode";
+
+import { GenericCommand } from "../src/runtime/lib/API";
+
+/**
+ * Namespace for common variables used globally in the extension.
+ * All variables here must be initialized in the activate() method of extension.ts
+ */
+export namespace ext {
```

Comment:
Any reason why we are only storing references for commands and not all other resources? Like webviews, tree views etc?

Reply:
Yeah, will add them in my next PR for adding tests for trees and webViews.